src/exodus/wallet.h

```diff
@@ -34,6 +35,7 @@ class Wallet
 
         return output;
     }
+    void ResetState();
```

Comment:
Right now this one will get invoked every time when we send tokens. My idea is we should remove this for now and handle it in #712 instead.